### DISCUSSION TITLE
Where to import focus-visible plugin?

### DISCUSSION THREAD

**Discussion by paulwongx** [tailwindlabs/tailwindcss#4554] [Help]
Documentation Link: https://tailwindcss.com/docs/hover-focus-and-other-states#focus-visible

In order to use focus-visible across all browsers, it suggests to download these two plugins.
1) https://github.com/WICG/focus-visible
2) https://github.com/csstools/postcss-focus-visible

#2 is a postcss plugin so I have included that in my craco.config.js under postcss: { plugins: [ <inserted_here> ] }
Where do I include plugin #1?

**Comment by VicGUTT** [comment]
Hey !

For the first one, as it is a standard JavaScript library that has nothing to do with PostCSS nor Tailwind, you can treat it as any other JavaScript library installed from NPM.

The [usage](https://github.com/WICG/focus-visible#usage) hints at requiring the script directly in your HTML: `<script src="/node_modules/focus-visible/dist/focus-visible.min.js"></script>`.

You could also maybe bundle it up with the reste of your app's JS:

```js
// app.js

import focusVisible from '...';
```
